trie/database.go

```diff
@@ -70,6 +70,10 @@ var (
 type Database struct {
 	diskdb ethdb.KeyValueStore // Persistent storage for matured trie nodes
 
+	// zktrie related stuff
+	Zktrie     bool
+	rawDirties KvMap // used only in ZkTrie. It's a quick&dirty implementation. FIXME later.
```

Comment:
add a `TODO` tag here.

```suggestion
        // TODO: It's a quick&dirty implementation. FIXME later.
	rawDirties KvMap // Used only in ZkTrie. 
```

Reply:
fixed